src/net/sf/mpxj/primavera/PrimaveraReader.java

```diff
@@ -692,33 +863,39 @@ private void addTaskUDFValue(Task task, Row row)
                break;
             }
 
+            case BOOLEAN:
+            {
+               value = STATICTYPE_UDF_MAP.get(row.getString("udf_text"));
```

Comment:
I'd prefer to leave the original values here, that way the application using the library can choose how to interpret them.